.github/workflows/test-release.yml

```diff
@@ -15,14 +15,14 @@ jobs:
       - uses: actions-rs/cargo@v1
         with:
           command: test
-          args: --release --no-default-features
+          args: --release --no-default-features --features "defmt"
```

Comment:
could you duplicate this so that we have both a configuration with defmt and one without?

Reply:
Should I do that for the debug CI as well?